app/scripts/museum/museum-service.js

```diff
@@ -100,6 +106,30 @@
             return Util.makeRequest(sql, query);
         }
 
+        function listByCity(params) {
+            var attributes = { 'gcity': 'city', 'gstate': 'state', 'gzip': 'zip' };
```

Comment:
What I mean is, if it's meant to filter by city, isn't filtering by zip too granular?